operate/webapp/src/main/java/io/camunda/operate/webapp/security/auth/ConsolidatedAuthenticationUserService.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.operate.webapp.security.auth;
+
+import io.camunda.authentication.entity.CamundaPrincipal;
+import io.camunda.operate.webapp.rest.dto.UserDto;
+import io.camunda.operate.webapp.security.Permission;
+import java.util.List;
+import org.springframework.context.annotation.Profile;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Component;
+
+@Component
+@Profile("consolidated-auth")
+public class ConsolidatedAuthenticationUserService extends AuthUserService {
+
+  @Override
+  public UserDto createUserDtoFrom(final Authentication authentication) {
+    if (authentication.getPrincipal() instanceof final CamundaPrincipal camundaPrincipal) {
+      return new UserDto()
+          .setUserId(camundaPrincipal.getPrincipalName())
+          .setDisplayName(camundaPrincipal.getDisplayName())
+          .setCanLogout(true)
+          .setPermissions(List.of(Permission.READ, Permission.WRITE));
```

Comment:
❓ Are these permissions used for just FE? Or do they also control anything else in the V1 APIs?

Reply:
I have no idea :D I just kept it as it was, hoping that if it worked for basic it should work for oidc as well.

Reply:
Hmm, I think we need to check this because when I check the code the `UserDto#getPermissions` method seems to be called in the security root [here](https://github.com/camunda/camunda/blob/main/operate/webapp/src/main/java/io/camunda/operate/webapp/security/CustomSecurityExpressionRoot.java#L30) which is then used to check the V1 APIs permission access. I think this could mean the V1 APIs act differently to the V2 APIs

Reply:
I see, so then it can be another task to use the same permission checking mechanism for both v1 and v2, as this PR scoped to oidc changes only and as it already was working with basic auth, it behaves same for oidc. wdyt?

Reply:
Yea I think it could be a separate task, I raised it to @ThorbenLindhauer as it could be something that we just accept for alpha5 to make progress and adjust after.